src/admin-views/admin-welcome-message.php

```diff
@@ -1,56 +1,73 @@
+			<li><a href="http://m.tri.be/1aa4" target="_blank"><?php esc_html_e( 'Settings Overview', 'the-events-calendar' ); ?></a></li>
+			<li><a href="http://m.tri.be/1aa5" target="_blank"><?php esc_html_e( 'Themer\'s Guide', 'the-events-calendar' ); ?></a></li>
+		</ul>
 	</div>
+
 	<div class="tribe-half-column">
-		<h2><?php esc_html_e( 'Newsletter Signup', 'the-events-calendar' ); ?></h2>
-		<p><?php esc_html_e( 'Stay in touch with The Events Calendar team. We send out periodic updates, key developer notices, and even the occasional discount.', 'the-events-calendar' ); ?></p>
-		<form action="https://moderntribe.createsend.com/t/r/s/athqh/" method="post">
-			<p><input id="listthkduyk" name="cm-ol-thkduyk" type="checkbox" /> <label for="listthkduyk"><?php esc_html_e( 'Developer News', 'the-events-calendar' ); ?></label></p>
-			<p><input id="listathqh" name="cm-ol-athqh" checked type="checkbox" /> <label for="listathqh"><?php esc_html_e( 'News and Announcements', 'the-events-calendar' ); ?></label></p>
-			<p><input id="fieldEmail" class="regular-text" name="cm-athqh-athqh" type="email" placeholder="<?php esc_attr_e( 'Email', 'the-events-calendar' ); ?>" required /></p>
-			<button type="submit" class="button-primary"><?php esc_html_e( 'Sign Up', 'the-events-calendar' ); ?></button>
-		</form>
+		<h2 data-tribe-icon="dashicons-sos"><?php esc_html_e( 'Resources and Support', 'the-events-calendar' ); ?></h2>
+		<p><?php esc_html_e( 'We’ve got your back every step of the way.', 'the-events-calendar' ); ?></p>
+		<ul>
+			<li><a href="http://m.tri.be/1aa6" target="_blank"><?php esc_html_e( 'Search the Knowledgebase', 'the-events-calendar' ); ?></a></li>
+			<li><a href="http://m.tri.be/1aa7" target="_blank"><?php esc_html_e( 'Available Translations', 'the-events-calendar' ); ?></a></li>
+			<li><a href="http://m.tri.be/1aa8" target="_blank"><?php esc_html_e( 'Submit a Help Desk Request', 'the-events-calendar' ); ?></a></li>
+		</ul>
 	</div>
 </div>
 
-<hr/>
+<div class="tribe-row">
 
-<div class="tribe-row tribe-welcome-links">
 	<div class="tribe-half-column">
-		<h4><?php esc_html_e( 'Getting Started', 'the-events-calendar' ); ?></h4>
-		<p><a href="https://m.tri.be/no" target="_blank"><?php esc_html_e( 'Check out the New User Primer &amp; Tutorials', 'the-events-calendar' ); ?></a></p>
-
-		<h4><?php esc_html_e( 'Looking for More Features?', 'the-events-calendar' ); ?></h4>
-		<p><a href="https://m.tri.be/np" target="_blank"><?php esc_html_e( 'Addons for Community, Tickets, Filters, and more.', 'the-events-calendar' ); ?></a></p>
-
-		<h4><?php esc_html_e( 'Support Resources', 'the-events-calendar' ); ?></h4>
-		<p><a href="https://m.tri.be/nq" target="_blank"><?php esc_html_e( 'FAQs, Documentation, Tutorials and Forums', 'the-events-calendar' ); ?></a></p>
+		<h2><?php esc_html_e( 'The Latest and Greatest', 'the-events-calendar' ); ?></h2>
```

Comment:
There is a lot of switching from `<h4>` to `<h2>` through here (and in the other PRs?) - is this format/size or context/syntactical?

Reply:
Mainly for formatting/size